src/style.css

```diff
@@ -1 +1,106 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+:root{
+  --text-color: #3D4E61;
+}
+
+body {
+  color: var(--text-color);
+}
+
+.search-big {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  margin-top: 20px;
+}
+
+.search-big_label {
+  position: relative;
+  width: 100%;
+}
+
+.search-big_label::after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  position: absolute;
+  content: "";
+}
+
+.search-big_input {
+  width: 100%;
+  height: 100%;
+  padding-left: 62px;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  outline: none;
+}
+
+.search-big_input:focus {
+  background-image: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+  box-shadow: 0 4px 4px 0 #00000040;
+  border-radius: 3px;
+}
+
+.search-big_input:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search-small {
+  width: 100%;
+  height: 42px;
+  display: flex;
+  margin-top: 20px;
+}
+
+.search-small_label {
+  position: relative;
+  width: 100%;
+}
+
+.search-small_label::after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  position: absolute;
+  content: "";
+}
+
+.search-small_input {
+  width: 100%;
+  height: 100%;
+  padding-left: 33px;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  outline: none;
+}
+
+.search-small_input:focus {
+  background-image: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+  box-shadow: 0 3px 8px 0 #3D4E6133;
+}
+
+.search-small_input:hover {
+  box-shadow: 0 3px 8px 0 #3D4E6133;
+}
```

Comment:
You have a lot of duplicated styles, the idea here is to use main class for both big and small forms and use modifier class to change some styles for small one